codewof/programming/content/en/leap-year/question.md

```diff
@@ -0,0 +1,6 @@
+# Leap Year
+
+Write a function `is_leap_year(year)` that takes a year and **returns** `True` if `year` is a leap year or `False` if it is not. The following rules determine whether a year is a leap year:
+
+- A leap year must be evenly divisible by 4.
+- A year that is evenly divisible by 100 is *only* a leap year if it is also evenly divisible by 400.
```

Comment:
Could we add a note about how % would be useful for finding out if a number is divisible by another number?

Reply:
Done